### PR TITLE
1 ajaxify comments likes and follow requests

### LINKED ISSUE #1: Ajaxify comments, likes, and follow requests

### PR DESCRIPTION
**Pull request by MauritzKurt**

### CODE REVIEW

**Review by JoseMDO — APPROVED** [review]
Everything looks good, great job Kurt!

File: app/controllers/comments_controller.rb
Comment (rayguna) [inline app/controllers/comments_controller.rb:1]: A concise Ajax request call using just `format.js`, consistently used throughout.

File: app/views/follow_requests/_follow_unfollow.html.erb
Comment (rayguna) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: A concise use of the class selector.

File: app/views/follow_requests/create.js.erb
Comment (rayguna) [inline app/views/follow_requests/create.js.erb:1]: A neat use of clearly labeled variables throughout makes the code readable.

**Review by rayguna — APPROVED** [review]

**Pull request merged**